pkgs/servers/isso/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, buildPythonApplication
+, misaka, werkzeug, ipaddr, configparser, html5lib_0_9999999 }:
+
+let
+  misaka_1 = misaka.overridePythonAttrs (old: rec {
+    version = "1.0.2";
+    name = "${old.pname}-${version}";
```

Comment:
`pname`

Reply:
I don't understand this. Should I redefine pname like this and use pname instead of old.pname?
```
pname = old.pname;
```

Or should I omit redefining `name` entirely?

Reply:
If you leave out `name` in the misaka expression, there is no need to override any of `name` or `pname`.